app/models/legal_aid_application.rb

```diff
@@ -28,6 +28,8 @@ class LegalAidApplication < ApplicationRecord
 
   delegate :full_name, to: :applicant, prefix: true, allow_nil: true
 
+  scope :latest, -> { order(created_at: :desc) }
```

Comment:
Nice. I've wondered about adding a `default_scope` to set the ordering based on created at, but that can be problematic, so this is probably the best compromise.